apis/addons/v1alpha1/addons_types.go

```diff
@@ -31,6 +36,10 @@ type AddonInstallSpec struct {
 type AddonInstallCommon struct {
 	// Namespace to install the Addon into.
 	Namespace string `json:"namespace"`
```

Comment:
Requires `// +kubebuilder:validation:MinLength=1` annotation, see comment abolut DisplayName.
(Not your bug, but would be very nice to fix within this PR)

Reply:
Done :)